tox.ini

```
@@ -48,10 +48,9 @@ commands =
     coverage combine
     coverage html
     coverage xml
-    coverage report --skip-covered --show-missing --fail-under=100.0
+    coverage report --show-missing --fail-under=100.0
```

I do not understand why you want to see a long list of 100 % covered modules as 100 % coverage is required. If a module drops below it will be shown as the only one with too low coverage no need to search for the module in a long output.

```suggestion
    coverage report --skip-covered --show-missing --fail-under=100.0
```

---

well I do understand that you like to only show the files where actual coverage is missing. 
I have reactivate that setting but in the setup.cfg file

---

Nice idea.